deploy/docker-compose-single.yml

```diff
@@ -26,7 +26,7 @@ services:
       - d3-network
 
   d3-changelog:
-    image: nexus.iroha.tech:19002/d3-deploy/changelog-endpoint:master
+    image: nexus.iroha.tech:19004/soranet/changelog-endpoint:master
```

Comment:
its not yet there

Reply:
If you want to use two registry in tests you have to add the step to login for both.